packages/turf-clone/index.js

```diff
@@ -70,12 +77,21 @@ function cloneFeature(feature) {
  * @returns {FeatureCollection<any>} cloned Feature Collection
  */
 function cloneFeatureCollection(geojson) {
-    return {
-        type: 'FeatureCollection',
-        features: geojson.features.map(function (feature) {
-            return cloneFeature(feature);
-        })
-    };
+    const cloned = {type: 'FeatureCollection'};
+    // Add id & bbox first
+    if (geojson.id) cloned.id = geojson.id;
+    if (geojson.bbox) cloned.bbox = geojson.bbox;
+
+    // Custom Properties
+    Object.keys(geojson).forEach(function (key) {
+        if (['id', 'type', 'bbox', 'features'].indexOf(key) !== -1) return;
+        cloned[key] = geojson[key];
```

Comment:
Sure, no impact on _that_ specific issue; I just used that as an example of creating a reference to the item instead of clone it.